src/main/java/Task.java

```diff
@@ -0,0 +1,21 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected static int tasksCount = 0;
```

Comment:
Structure code logically

`tasksCount` should be in the same place as Task array, may cause confusion to reader